### ISSUE TITLE
Add WHERE clause support for SHOW TOPICS

### ISSUE THREAD

**Issue by swamigee** [confluentinc/ksql#1459]
Where clause for show/list topics command will be really useful when listing hundreds of topics.
e.g. 

    show topics where topic-name='test.*'

Currently this lists 

    Kafka Topic   | Registered | Partitions | Partition Replicas | Consumers | Consumer Groups

May be more values could be listed. 
e.g. 
**offsets**
**metrics**

**Comment by rmoff** [comment]
Would we do this as part of the `SHOW TOPICS` syntax, or maybe bring in a concept of system tables (as in RDBMS) so that more advanced queries like this one can be done with proper SQL, e.g. 

    SELECT TOPICS FROM SYS.TOPICS WHERE TOPIC_NAME LIKE 'TEST%';

**Comment by swamigee** [comment]
Prefer SYS as you suggested.

**Comment by blueedgenick** [comment]
was thinking the same thing - this is the start of implementing the whole idea of treating all the management and metadata structures as if they were tables (thank you Dr. Codd!). 

Ahh, how I miss my `V$...` views....

**Comment by spena** [comment]
Other DBs support the `like` keyword in the `show` statement. Perhaps using a similar keyboard would be better for DBs users comming to ksqlDB. 

This should be useful to search for a specific stream, table or topic, especially in when using the extended keyword. Currently, `show streams extended` displays lots of information for all existing streams and tables which makes it look ugly. +1 on having a keyboard to filter the objects.
